assets/app/features/upgrade/templates/upgrade7/database-configuration.jade

```diff
@@ -0,0 +1,49 @@
+h2(translate='') upgrade7.steps.database-configuration.title
+
+uib-tabset(active='active')
+  uib-tab(index='0', heading='Create a Database')
+    form(novalidate, name='createDatabase')
+      .form-group(ng-class="{'has-warning': createDatabase.username.$error.required}")
+        .col-lg-8.col-xs-12 
+          .label-control
+            label(translate='') upgrade7.steps.database-configuration.databaseForm.username
+            span(translate='', ng-show='createDatabase.username.$error.required') - invalid characters
```

Comment:
I have removed code related to validation, because in this branch we are not working on validation part.